### ISSUE TITLE
Internal Server Hang on Startup (1.19.2, MFFS-5.0-alpha2.7)

### ISSUE THREAD

**Issue by weirdspecter** [BuiltBrokenModding/MFFS_Classic#47]
No crash log to include because of the nature of the issue, but I spent the morning tracking down an issue I've never experienced before in a modpack, which vanished once I removed MFFS-5.0-alpha-2.7 from the pack.

Upon opening an existing minecraft world, there was massive internal server lag. I could move around, had no issues with framerate lag or microstutter, but it was as if the singeplayer game's internal server couldn't keep up. Chat messages and commands sent in the chat window would never appear (and, checking the log, were never received by the internal server), and other aspects of the world loaded incorrectly. The first time I'd opened the world with MFFS installed, yesterday, there had been no issues at all.

If it's relevant, there was an MFFS setup within render distance.

**Comment by Su5eD** [comment]
sounds like a TPS issue, how large was the projected forcefield?

**Comment by weirdspecter** [comment]
On the order of about 64 blocks, I think -- although it's worth noting that this didn't happen when the field was turned on or off last night, or when various upgrade cards were exchanged. It also didn't happen when the chunks containing the forcefield were loaded or unloaded.

The problem only presented itself on reloading the world.

**Comment by Su5eD** [comment]
I've tracked down the issue to force field selection, which checks every single block position to make sure it can be replaced by a field block. Running this for tens of thousands of blocks caused heavy lag. This logic has now been moved off-thread so it won't lag the server anymore.  
Can you check if the problem still persists in the latest version (`1.19.2-5.0-alpha.39`)?

**Comment by weirdspecter** [comment]
Hi there, just now tested it on the same world.

The world itself loads without internal sever lag, but there are now world holes randomly interspersed in the map, including in the forcefield's location. These weren't just graphical glitches, unfortunately, but "real" holes -- I could see the sides of neighbouring chunks through them.

They appeared, albeit less frequently, in new worlds I generated as well. I've confirmed that the issue is not present when MFFS is removed.

I'm really sorry, it doesn't seem to have worked. Or, if it did, it's only created a new problem.
